other/APIs/client_secret.bcheck

```diff
@@ -0,0 +1,63 @@
+metadata:
+    language: v2-beta
+    name: "Use of Known API Variable"
+    description: "This BCheck looks for known API variables (client_secret, client_id, refresh_token, & APIKey)."
+    author: "Kyle Gilligan"
+    tags: "passive", "API", "client_secret", "client_id", "ApiKey"
+
+define:
+    # Issue details (for discovery of insecure API variables) as individual string texts.
+    issueDetail1 = `A known sensitive API parameter has been discovered within a front-end file of this web`
+    issueDetail1a = `application.`
+    issueDetail1FULL = `{issueDetail1} {issueDetail1a}`
+    iD_clientSecret1 = `\n    • client_secret: This OAuth variable is used to authenticate applications towards`
+    iD_clientSecret2 = `an intended authorization server. Essentially the 'password' of API authorization.`
+    iD_clientSecret3 = `If a user finds both an API's client_id & client_secret, they may be able to access an`
+    iD_clientSecret4 = `endpoint's resources by impersonating the application.`
+    iD_clientSecretFULL = `{iD_clientSecret1} {iD_clientSecret2} {iD_clientSecret3} {iD_clientSecret4}`
+    iD_refreshToken1 = `\n    • refresh_token: Secret variables used by an application to request new access`
+    iD_refreshToken2 = `tokens for an API.`
+    iD_refreshTokenFULL = `{iD_refreshToken1} {iD_refreshToken2}`
```

Comment:
These (ID_refreshToken...) are probably simple enough to merge into a single variable (no need for interpolated strings then as well).

Reply:
Made changes based on recommendation.